tomcat/src/main/resources/static/login.html

```diff
@@ -20,7 +20,7 @@
                                 <div class="card shadow-lg border-0 rounded-lg mt-5">
                                     <div class="card-header"><h3 class="text-center font-weight-light my-4">로그인</h3></div>
                                     <div class="card-body">
-                                        <form method="get" action="login">
+                                        <form method="post" action="login">
```

Comment:
오호! 저도 이 부분이 왜 `GET `요청일까 의문을 가졌었는데 `HTML` 에서 수정은 선택하셨군요! 👍

`form` 태그는 기본적으로 `POST` 요청을 보내기 때문에 생략이 가능합니다!
```suggestion
                                        <form action="login">
```

Reply:
오!! 처음 안 사실인데, post를 생략해 주어도 되겠군요.

Reply:
헉!! 제가 반대로 알고 있었군요 ㅠㅠ
혼동을 일으켜 죄송합니다... 😥

https://www.w3.org/TR/html401/interact/forms.html#h-17.3
> method = get|post [[CI]](https://www.w3.org/TR/html401/types.html#case-insensitive)
This attribute specifies which HTTP method will be used to submit the [form data set](https://www.w3.org/TR/html401/interact/forms.html#form-data-set). Possible (case-insensitive) values are **"get" (the default)** and "post". See the section on [form submission](https://www.w3.org/TR/html401/interact/forms.html#submit-format) for usage information.